app/views/events/index.html.erb

```diff
@@ -0,0 +1,42 @@
+
+<div class="row">
+  <div class="col-md-6 form-new-user col-md-offset-3">
+  <h3> All Events </h3>
+    <ul>
+    <% @events.each do |event| %>
+        <li><h3><%= link_to event.title, event_path(event) %></h3></li>
+        <li><%= event.description %></li>
+        <li><%= event.location %></li>
```

Comment:
[OPTIONAL] consider using partials for this.

Reply:
fixed by using partial `render 'events/event'`